classic/main.tex

```diff
@@ -250,9 +249,11 @@
 \textcolor{softcol}{\hrule}
 \vspace{6pt}
 
-  $\cdot$ #4\\[3pt]
-  $\cdot$ #5\\[6pt]
-
+	\foreach \desc in {#4}{
```

Comment:
Hey @FFY00 thanks a lot, that's great! Since you did not import any new package I am curious of which package \forEach is part of.

Reply:
It is provided by TikZ :blush: